gulp-tasks/workbox/workbox-generate-contributors.js

```diff
@@ -14,7 +16,7 @@ const MAX_COLLABORATORS = 20;
  */
 
 gulp.task(`workbox-generate-contributors`, function() {
-  var token = process.env.GITHUB_TOKEN;
+  let token = process.env.GITHUB_TOKEN;
```

Comment:
we already have a `GIT_TOKEN`, how is this one different?

Reply:
This wasn't caught on first review of this code after you request the text file change. Fixed now.

Reply:
Yah, I realized that afterwards. It's the Git token for @WebFundBot. We should prob rename them at some point.